CHANGELOG.md

```diff
@@ -0,0 +1,101 @@
+# 0.3.0
+
+This version roughly corresponds with all the changes made to the Temporal polyfill as of the October 2021 TC39 Plenary meeting.
+
+Breaking changes:
+
+- Timezones now require a `getOffsetNanosecondsFor` method, and no longer fall back to the intrinsic definition (previously provided by `Temporal.Timezone#getOffsetNanosecondsFor`) if not provided. ([08346dc5])
```

Comment:
Is there a corresponding proposal-temporal PR? OK if not, just wanted to make sure.

Reply:
There is a corresponding PR, but weirdly it doesn't change the spec text. Seems maybe the spec text was already  this way (for a while): https://github.com/tc39/proposal-temporal/blame/60c753a6369c0a61cacc353562214aaa6327117a/spec/timezone.html#L586-L597